src/uk/gov/defra/ffc/DefraUtils.groovy

```diff
@@ -7,6 +7,14 @@ def repoUrl = ''
 def commitSha = ''
 def workspace
 
+def getCSProjVersion(projName) {
+  def version = sh(returnStdout: true, script: "xmllint $projName/$projName\.csproj --xpath '//Project/PropertyGroup/Version/text()'").trim()
+  if (version == '') {
+    version = sh(returnStdout: true, script: "xmllint $projName/$projName\.csproj --xpath 'concat(//Project/PropertyGroup/VersionPrefix/text(), \"-\", //Project/PropertyGroup/VersionSuffix/text())'").trim()
```

Comment:
Looks like the version structure used in our project is the `PropertyGroup/Version`, one. 
Could the code be simplified to be the only the first bit of logic if we standardised on how the version is set in the csproj files?

Reply:
It absolutely could - I was trying to make it a bit more bulletproof, seemingly there is no guarantee that either Version or VersionPrefix / VersionSuffix will be set, having created a new dotnet core console project and had a look at the csproj file it generated. I'll check with the dot net guys

Reply:
Here's my comments from Slack so we have a rationale for the decision in one place.

I agree we should settle on a standard of using the `Version`.  We have no need identified for suffixes or prefixes with our version strategy so let's not add complexity we can avoid.  If there is a future need we can revisit.  This standard should also be added to the dev guide but we shouldn't increase the scope of this ticket.  Creating a new ticket for that update is fine.